packages/wonder-blocks-tokens/src/tokens/media-queries.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * A default set of media queries to use for different screen sizes.
+ *
+ * Breakpoint documentation: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2099970518/Layout+Breakpoints
+ *
+ */
+
+const width = {
+    xsMax: 567,
+    smMin: 568,
+    smMax: 681,
+    mdMin: 682,
+    mdMax: 1023,
+    lgMin: 1024,
+} as const;
+
+const mediaQuery = {
+    // Named
```

Comment:
There should probably be a comment here telling people if they make changes here they need to also update aphrodite.d.ts.